MM-31674 Cloud app/product notices for ext dependecies (#4418)
mattermost/docs: source/cloud/cloud-administration/site-configuration.rst

Administrator Notices
^^^^^^^^^^^^^^^^^^^^^

> Administrator notices inform System Admins when a new server version is available, when external dependencies are being deprecated, or when a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.

Reviewer: ```suggestion
System Admins may receive notices about recommended configuration options to optimize the user experience of their workspace.
```

```
